components/tabs/src/storage.rs

```diff
@@ -558,7 +598,8 @@ impl TabsStorage {
     }
 
     // Remove any pending tabs that are 24hrs older than the last time that client has synced
-    // Or that client's incoming tabs does not have those tabs anymore
+    // Or that client's incoming tabs reflects that the command was acted on (eg, for tab close
```

Comment:
Zepto-nit: Sentence fragment 👓 and missing triple-slash comment. Let's do:

```rs 
/// Removes any pending "close tabs" commands that are either:
///
/// 1. 24hrs older than the last time the target device has synced, or...
/// 2. The target device's incoming tabs reflect that the command was acted on
///    (eg, for tab close commands; does not have those tabs anymore;
///    for close inactive commands, has no inactive tabs).
```

(I know this was already there before; sorry to ask you to fix it!)